### PR TITLE
respin latest version to include 25.0 [release]

### PR DESCRIPTION
**Pull request by JalexChen**

### CODE REVIEW

File: push-images.sh [hunk 1]
```diff
@@ -14,13 +14,19 @@ docker push cimg/elixir:1.13-erlang-23.3-browsers
...
+docker tag cimg/elixir:1.13.4-erlang-25.0 cimg/elixir:1.13.4
+docker tag cimg/elixir:1.13-erlang-25.0 cimg/elixir:1.13
+docker push cimg/elixir:1.13-erlang-25.0-node
+docker push cimg/elixir:1.13.4-erlang-25.0-node
+docker tag cimg/elixir:1.13.4-erlang-25.0-node cimg/elixir:1.13.4-node
+docker tag cimg/elixir:1.13-erlang-25.0-node cimg/elixir:1.13-node
+docker push cimg/elixir:1.13-erlang-25.0-browsers
+docker push cimg/elixir:1.13.4-erlang-25.0-browsers
+docker tag cimg/elixir:1.13.4-erlang-25.0-browsers cimg/elixir:1.13.4-browsers
+docker tag cimg/elixir:1.13-erlang-25.0-browsers cimg/elixir:1.13-browsers
```
Comment (felicianotech) [inline push-images.sh:32]: Does this have the latest shared-tools? It's missing the lines where we assume the parent tag. For example, something like this:

docker push cimg/elixir:1.13

File: 1.13/25.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+FROM cimg/base:2022.01
```
Comment (felicianotech) [inline 1.13/25.0/Dockerfile:5]: Base image wasn't updated.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: push-images.sh (hunk 1 above)
Comment (JalexChen) [reply push-images.sh:32]: oh gosh. i think i only did it for the alias and not the default parent tag

Comment (JalexChen) [reply push-images.sh:32]: this is the PR for it. i'll respin after this has been merged and updated on this repo: https://github.com/CircleCI-Public/cimg-shared/pull/67

**Review by felicianotech — APPROVED** [review]

**Pull request merged**